microprot/scripts/processing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Tomasz Kosciolek"
+__version__ = "1.01b"
+__last_update__ = "12/19/2016"
```

Comment:
Agreed. I will change if @tkosciol also agrees and decides to change the style of the entire project. Otherwise I will just remain consistent.